recipes/uldaq/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "uldaq" %}
+{% set version = "1.2.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 5cfc152a67381717bcf6f275f4f059c7adfae830eb4dfca35d90972a6021f84c
+  patches:
+    - 0001-ul_c_interface-use-ctypes.util.find_library.patch
+
+build:
+  number: 0
+  noarch: python
+
+requirements:
+  build:
+    - pip
+    - python
+  host:
+    - libuldaq
+    - pip
+    - python
+    - {{ cdt('libudev') }}
+  run:
+    - libuldaq
+    - python
+    - {{ cdt('libudev') }}
```

Comment:
```suggestion
  host:
    - pip
    - python
  run:
    - libuldaq
    - python
```

Reply:
Why are you suggesting that this dependency be removed?

Reply:
http://conda-forge.org/docs/maintainer/knowledge_base.html?#core-dependency-tree-packages-cdts
"CDT" packages are those that we use when no equivalent Conda package is available. Those are then expected to be provided by the system at runtime. You only ever want `{{ cdt(...) }}` in the `requirements: build` section, if at all.
Having them in `host`/`run` requirements won't work because they are installed into a `x86_64-conda-linux-gnu/sysroot/` folder (which will be picked at build time), i.e., separate from what your packages will see/use.
Since this is a pure Python package, you don't need the CDT one in `requirements: build` either, because the build doesn't need it (e.g., it's not linking to the library).If `libudev` is required at run time, you should add a `yum_requirements.txt` for that, cf. https://github.com/conda-forge/libiio-feedstock/blob/master/recipe/yum_requirements.txt .

Reply:
Thanks for the explanation! I'll try that and see how it goes :)

Reply:
Thanks for the explanation! I'll try that and see how it goes :

Reply:
The `yum_requirements.txt` from your last commit has `libusb`. Did you mean to use `libusb` or `libudev`?

Reply:
Thanks, it should be `libudev`.

Reply:
I'm not sure what to do about the failure to build on windows. As mentioned in a comment above, this library is only for osx and linux, so it makes sense that it wouldn't build on windows.

Reply:
Originally, I had
```
build:
    skip: true  # [win]
```

Reply:
Well, I tried it with the `skip` above and everything seems to pass -- awesome! I'll squash these fixups and hopefully everything will be good. Thanks for the help @mbargull :)

Reply:
> Well, I tried it with the `skip` above and everything seems to pass

I'm fine with either approach here.
The background is that Conda has special handling for pure Python packages with this `noarch: python` type for which we can have one build that can be served on all platforms for all Python versions. However, we don't have a good(/established) way to declare if a `noarch: python` package is not compatible with some systems (e.g., Windows here). Hence, we could either ignore the build failure and let `conda`'s solver figure out when dependencies aren't available (which sometimes/often result in hard-to-read package conflict messages for the user) or just duplicate the build on compatible systems/Python versions (i.e., this is what `skip: true  # [win]` gives you).

> Thanks for the help @mbargull :)

Happy to!